src/video-recorder/index.js

```diff
@@ -58,7 +58,10 @@ export default class VideoRecorder {
     }
 
     _assignEventHandlers (browserJob) {
-        browserJob.once('start', this._createSafeListener(this._onBrowserJobStart));
+        browserJob.once('start', this._createSafeListener(() => {
```

Comment:
Please add the `async` modifier and add `await this.tempDirectoryInitializedPromise` OR return `this.tempDirectoryInitializedPromise` from the handler.